packages/bottom-tabs/src/types.tsx

```diff
@@ -326,3 +334,13 @@ export type BottomTabBarButtonProps = Omit<
     e: React.MouseEvent<HTMLAnchorElement, MouseEvent> | GestureResponderEvent
   ) => void;
 };
+
+export type TabBarPositionMapConfig = Record<
+  TabBarPosition,
+  Record<'tabBar' | 'content' | 'tabBarItem', StyleProp<ViewStyle>>
+>;
+
+export type TabBarContainerPositionMapConfig = Record<
+  TabBarPosition,
+  StyleProp<ViewStyle>
+>;
```

Comment:
These types are very specific to the logic inside the tab bar, so they should be located where they are used

Reply:
Got you